curvefs/src/mds/schedule/copysetSchedule.cpp

```diff
@@ -222,8 +222,62 @@ int CopySetScheduler::CheckAndBalanceZone(ZoneIdType zoneId) {
             if (!IsCopysetCanTransfer(copyset)) {
                 continue;
             }
-            LOG(INFO) << "transfer copyset from metaserver " << sourceId
-                      << " to metaserver " << destId;
+            LOG(INFO) << "transfer copyset from high usage metaserver "
+                      << sourceId << " to low usage metaserver " << destId;
+            if (TransferCopyset(copyset, sourceId, destId)) {
+                genOpCount++;
+                return genOpCount;
+            }
+        }
+    }
+
+    return genOpCount;
+}
+
+int CopySetScheduler::CheckAndBalanceZoneByCopysetNum(ZoneIdType zoneId) {
```

Comment:
CheckAndBalanceZoneByCopysetNum and CheckAndBalanceZoneByMetaserverUsage are mainly different in sort function, so you can think can they combined into one.

Reply:
these two has sequence, by usage first, then by copysets num.
If usage has a operate, then skip the copysets num balance.